packages/shared/tsconfig.json

```diff
@@ -12,7 +12,8 @@
             "@core/*": ["./lib/core/*"],
             "@features/*": ["./features/*"],
             "@lib/*": ["./lib/*"]
-        }
+        },
+        "types": ["node"]
```

Comment:
for what is this needed?

Reply:
I was having some weird compilation errors regarding types and I couldn't get it to compile. It was telling me to do this. Let me double check.

Reply:
Okay I have no idea why but we now need this to compile 🤷‍♀️